scripts/tools/eslint-plugin-prettier-internal-rules/no-node-comments.js

```diff
@@ -31,23 +30,28 @@ module.exports = {
     type: "suggestion",
     docs: {
       url:
-        "https://github.com/prettier/prettier/blob/main/scripts/eslint-plugin-prettier-internal-rules/no-node-comments.js",
+        "https://github.com/prettier/prettier/blob/main/scripts/tools/eslint-plugin-prettier-internal-rules/no-node-comments.js",
```

Comment:
Shouldn't this be fixed in upstream Prettier?

Otherwise ... yeah!!

Reply:
Yeah, it definitely belongs upstream. I left it in while testing so that I could tell when yarn/npm switched to using the updated copy, but I forgot to pull it out before committing.